src/styles/jeres.js

```diff
@@ -0,0 +1,215 @@
+// jeres
+// Status: "Ready"
+// Twitter: @heyjeres
+// Fxhash: https://www.fxhash.xyz/u/jeres
+// Wallet: tz1hnnCaYDiRPH8apoJQB9wvytwpYQSJEzCK
+
+import Style from './style'
+
+export default class JeresStyle extends Style {
+
+  WHITE = "#F4EEDC";
+  BLACK = "#45413C";
+  COLORS = ["#CEAC41", "#DAC27C", "#D7AF72", "#EFDEC2", "#A8651E"];
+  SKYCOLORS = ["#948f9f", "#3E92CC"];
+  WATERCOLORS = ["#2169BA", "#11489F"];
+  TILECOLORS = ["#c93b36", "#c85470", "#4c9784", "#41569a", "#948f9f"];
+
+  WIND = 0;
+
+  constructor(gridSizeX, gridSizeY, s, projectionCalculator3d, p5) {
+    super(gridSizeX, gridSizeY, s, projectionCalculator3d, p5)
+    console.log(`y: ${gridSizeY}`);
```

Comment:
You can remove all the console.log used for debug

Reply:
removed!